interfaces/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+
+# add_test(NAME matlab_test_sim_pendulum 
+#         COMMAND "${CMAKE_COMMAND}" -E chdir ${PROJECT_SOURCE_DIR}/examples/matlab_mex/pendulum_on_cart_model ${MATLAB_EXECUTABLE} -nodisplay -nodesktop -r  "run('./run_tests.m');")
+
+add_test(NAME octave_test_sim 
+        COMMAND "${CMAKE_COMMAND}" -E chdir ${PROJECT_SOURCE_DIR}/examples/matlab_mex/test
+        octave --no-gui --no-window-system ./run_tests_sim.m)
```

Comment:
does `octave-cli` do the same?

Reply:
On my machine, it does not make a difference if I write 
`octave` or `octave-cli`
Might depend on the installation.